Tests/YamsTests/EncoderTests.swift

```diff
@@ -348,6 +348,26 @@ class EncoderTests: XCTestCase { // swiftlint:disable:this type_body_length
         _testRoundTrip(of: date)
     }
 
+    func testDecoderMark() throws {
+        let s = """
+            Value
+            """
+        struct Test: Decodable {
+            var value: String
+            var location: String
+
+            init(from decoder: Decoder) throws {
+                let container =  try decoder.singleValueContainer()
+
+                self.value = try container.decode(String.self)
+                self.location = decoder.mark?.description ?? ""
+            }
+        }
+        let t = try YAMLDecoder().decode(Test.self, from: s)
+        XCTAssertEqual(t.value, "Value")
+        XCTAssertEqual(t.location, "1:1")
```

Comment:
I would have tested a mapping here, but #297 prevents the value from being what it should be.

Reply:
Can you rebase this PR after merging https://github.com/jpsim/Yams/pull/297 so you can add that test?

Reply:
:+1: I'll also add the linux change needed.